services/src/messaging/src/main/java/org/openkilda/messaging/command/flow/FlowRequest.java

```diff
@@ -15,27 +15,54 @@
 
 package org.openkilda.messaging.command.flow;
 
-import org.openkilda.messaging.Utils;
 import org.openkilda.messaging.command.CommandData;
-import org.openkilda.messaging.model.FlowDto;
+import org.openkilda.model.SwitchId;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+import lombok.NonNull;
 import lombok.Value;
 
 @Value
+@JsonNaming(value = SnakeCaseStrategy.class)
+@Builder(toBuilder = true)
 public class FlowRequest extends CommandData {
 
-    @JsonProperty(Utils.PAYLOAD)
-    private FlowDto payload;
+    @NonNull
```

Comment:
JsonProperty?

Reply:
@timofei-durakov it will be added by @JsonNaming